package-lock.json

```diff
@@ -6767,111 +7486,125 @@
         "node": ">= 12.0.0"
       }
     },
-    "node_modules/@aws-sdk/signature-v4-crt": {
-      "version": "3.52.0",
-      "resolved": "https://registry.npmjs.org/@aws-sdk/signature-v4-crt/-/signature-v4-crt-3.52.0.tgz",
-      "integrity": "sha512-+9b8WDHxCJ9OoXWMhV91Xt2nQtjqYatR2NA+gUUHjOiHbSSY0nG0jMsgKQU7MP6OAEWpZnRWEpeZtwwJm+gatg==",
+    "node_modules/@aws-sdk/signature-v4-multi-region": {
+      "version": "3.226.0",
+      "resolved": "https://registry.npmjs.org/@aws-sdk/signature-v4-multi-region/-/signature-v4-multi-region-3.226.0.tgz",
+      "integrity": "sha512-QHxNuf9ynK208v7Y3imdsa3Cz8ynYV7ZOf3sBJdItuEtHN6uy/KxaOrtvpF8I5Hyn48Hc8z5miTSMujFKT7GEw==",
       "dev": true,
-      "peer": true,
       "dependencies": {
-        "@aws-sdk/is-array-buffer": "3.52.0",
-        "@aws-sdk/querystring-parser": "3.52.0",
-        "@aws-sdk/signature-v4": "3.52.0",
-        "@aws-sdk/util-hex-encoding": "3.52.0",
-        "@aws-sdk/util-uri-escape": "3.52.0",
-        "aws-crt": "^1.9.7",
```

Comment:
As a side effect of the update of @aws-sdk/client-s3, this has been updated as well and does no longer include aws-crt as a direct dependency.